src/lib/customTraceType.js

```diff
@@ -40,6 +40,8 @@ export function traceTypeToPlotlyInitFigure(traceType, gl = '') {
       return {type: 'scatter' + gl, fill: 'tozeroy'};
     case 'scatterpolar':
       return {type: 'scatterpolar' + gl};
```

Comment:
are trace should default to NOT specifying fill and to having stackgroup='Group 1'

Reply:
in area recognition, we also need to check for a stackgroup